uaclient/apt.py

```diff
@@ -601,6 +653,12 @@ def remove_auth_apt_repo(
 ) -> None:
     """Remove an authenticated apt repo and credentials to the system"""
     system.ensure_file_absent(repo_filename)
+    # Also try to remove old .list files for compatibility with older releases.
+    # If repo_filename is already .list, this is just a 'noop'
```

Comment:
Noop, yes, but will trigger a log warning, right? About the file not being there, because the call above already removed it.

Reply:
It will be a double log, but not a warning

Reply:
Sorry, not a warning, but a debug log, which is the default log level of the client, right?

Reply:
I added a check before the removal just to make sure we only try to remove the `.list` when removing a `.sources`.
- for SRUs, this guarantees no double logging
- from Noble on the log that the file exists will be there, but we actually do want it to be there.